composer.json

```diff
@@ -19,7 +19,8 @@
         "symfony/property-info": "^6.4 || ^7.1",
         "symfony/serializer": "^6.4 || ^7.1",
         "symfony/type-info": "^6.4 || ^7.1",
-        "symfony/uid": "^6.4 || ^7.1"
+        "symfony/uid": "^6.4 || ^7.1",
+        "webmozart/assert": "^1.11"
```

Comment:
Not needed, but I usually use this lib to assert things in my value objects, we can ofc go with plain php if you want to avoid the dep

Reply:
yea, mixed but no hard feelings :D